File: Models/Message.cs
using System;

namespace Models
{
    [Serializable]
    public class Message
    {
        public string text;
        public int[] arrInt;
        public double[] arrDouble;

        public Message(string text, int[] arrInt) {
            this.text = text;
            this.arrInt = arrInt;
        }
        public Message(string text, double[] arrDouble) {
            this.text = text;
            this.arrDouble = arrDouble;
        }
       
    }
}

File: java_pkcp_2/src/Message.java
import java.io.Serializable;

public class Message implements Serializable {
    private String text;
    private int[] arrInt;
    private double[] arrDouble;

    public Message(String text, int[] arrInt) {
        this.text = text;
        this.arrInt = arrInt;
    }
    public Message(String text, double[] arrDouble) {
        this.text = text;
        this.arrDouble = arrDouble;
    }
    public String getText() {
        return text;
    }

    public double[] getArrDouble() {
        return arrDouble;
    }

    public int[] getArrInt() {
        return arrInt;
    }
}
